README.md

```diff
@@ -1,18 +1,30 @@
-## 👋 Welcome!
+# Conda GitHub Org Infrastructure
+
+This is a centralized location for infrastructure codes and any related inquiries or discussions.
+
+<table><tr><td><b>⚠️ This is not a forum for end-user questions.</td></tr></table>
 
-We’re using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place to provide a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
-  * Package hosting (e.g. conda-forge channel on anaconda.org)
-  * GitHub organization admin questions and issues
-  * Test suite and development setup issues
-  * Anything that does not fall into any other project specific buckets (e.g. the [conda issues](https://github.com/conda/conda/issues) or [conda-build issues](https://github.com/conda/conda-build/issues))
-  * etc.
+## 👋 Welcome!
 
-**NOTE: This is not a forum for end user questions!**
+We are using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
```

Comment:
That reminds me, that this repo isn't only for Conda core members but was originally created for all community members that want to raise issues around hosting etc, e.g. Bioconda and conda-forge maintainers. Let's fix this while we're here.

Reply:
Which phrasing?

1. "...Conda core members and community maintainers..."
2. "...Conda core members and community members..."
3. "...community maintainers..."
4. "...community members..."

I'm partial for either of the "community maintainers" options, I think "community members" is too broad.

Reply:
Let's go with "Conda core and community members" that should be broad enough, right?